frontend/src/Activity/Queue/RemoveQueueItemsModal.js

```diff
@@ -122,6 +128,19 @@ class RemoveQueueItemsModal extends Component {
               />
             </FormGroup>
 
+            {
+              blocklist &&
```

Comment:
Please convert to a ternary: `blocklist ? ... : null`, similar to `allPending` starting at line 100 above.